index.html

```diff
@@ -12,18 +12,107 @@
 
 <body>
 
+  <!-- Main Container Begin -->
   <div class="container">
 
+    <!-- Top Navigation -->
+    <div class="Nav-Top">
```

Comment:
Make sure to be consistent in your naming convention when naming classes. The recommended one is the kebab case where all letters are lowercase and spaces are hyphens (ex. `nav-top`) 😄